src/lib/source-handlers/azure/get-azure-orgname.ts

```diff
@@ -0,0 +1,9 @@
+export function getAzureOrgName(): string {
+  const azureOrgName = process.env.AZURE_ORG;
+  if (!azureOrgName) {
+    throw new Error(
+      `Please set the AZURE_ORG e.g. export AZURE_ORG='myAzureOrgName'`,
```

Comment:
as per chat in person we will not need this anymore as we will assume `org` in Azure is the same concept as org in Github or Group in Gitlab so we will expect this to come from the orgData file.

Reply:
Correct, will remove

Reply:
Done